contrib/firmware_packager/simple_client.py

```diff
@@ -110,6 +111,16 @@ def device_changed(client, device, progress):  # pylint: disable=unused-argument
     progress.device_changed(device.get_name())
 
 
+def modify_config(client, key, value):
+    """Use fwupd client to modify daemon configuration value"""
+    try:
+        client.modify_config(key, value, None)
+    except GLib.Error as glib_err:
+        print("%s" % glib_err)
+        sys.exit(1)
```

Comment:
I think the convention is to call the exception `e` and then to do something like `print(str(e))`